Pester.psm1

```diff
@@ -221,6 +230,85 @@ about_pester
     if ($EnableExit) { Exit-WithCode -FailedCount $pester.FailedCount }
 }
 
+function ResolveTestScripts
+{
+    param ([object[]] $Path)
```

Comment:
Should we rename `$Path` to something different to represent that it may contain both path and arguments? Maybe  create a separate Parameter set for it